### PR TITLE
spctl: add page

### PR DESCRIPTION
**Pull request by timphan92**
- [X] The page (if new), does not already exist in the repository.
- [X] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [X] The page has 8 or fewer examples.
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [X] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [X] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

### CODE REVIEW

File: pages/osx/spctl.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+> Utility for managing Gatekeeper in OS X
+> More information: <https://www.unix.com/man-page/osx/8/SPCTL/>.
+
+- Turn off gatekeeper:
+
+`spctl --master-disable`
+
+- Add a rule to allow an application to run (Labeling of rule is optional):
+
+`spctl --add --label "RuleName" {{path/to/file}}`
```
Comment (navarroaxel) [inline pages/osx/spctl.md:13]: We use `snake_case` for placeholders
```suggestion
`spctl --add --label "{{rule_name}}" {{path/to/file}}`
```

File: pages/osx/spctl.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# spctl
+
+> Manage the security assessment policy subsystem.
+> Utility for managing Gatekeeper in OS X
```
Comment (navarroaxel) [inline pages/osx/spctl.md:4]:
```suggestion
> Utility for managing Gatekeeper in macOS.
```

File: pages/osx/spctl.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+> More information: <https://www.unix.com/man-page/osx/8/SPCTL/>.
+
+- Turn off gatekeeper:
```
Comment (navarroaxel) [inline pages/osx/spctl.md:7]:
```suggestion
- Turn off Gatekeeper:
```

File: pages/osx/spctl.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+`spctl --master-disable`
+
+- Add a rule to allow an application to run (Labeling of rule is optional):
+
+`spctl --add --label "RuleName" {{path/to/file}}`
+
+- Turn on gatekeeper:
```
Comment (navarroaxel) [inline pages/osx/spctl.md:15]:
```suggestion
- Turn on Gatekeeper:
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]

**Review by blueskyson — APPROVED** [review]
Look good to me.

File: pages/osx/spctl.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+
+> Manage the security assessment policy subsystem.
+> Utility for managing Gatekeeper in macOS.
+> More information: <https://www.unix.com/man-page/osx/8/SPCTL/>.
+
+- Turn off Gatekeeper:
+
+`spctl --master-disable`
+
+- Add a rule to allow an application to run (Labeling of rule is optional):
```
Comment (navarroaxel) [inline pages/osx/spctl.md:11]:
```suggestion
- Add a rule to allow an application to run (labeling of rule is optional):
```

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**